tools/intersmash-tools-core/src/main/java/org/jboss/intersmash/tools/provision/openshift/operator/OperatorProvisioner.java

```diff
@@ -415,7 +415,8 @@ public void unsubscribe() {
 		adminBinary.execute("delete", "subscription", packageManifestName, "--ignore-not-found");
 		adminBinary.execute("delete", "csvs", currentCSV, "--ignore-not-found");
 		for (String customResource : getCustomResourceDefinitions()) {
-			if (adminBinary.execute("get", "crd", customResource, "--ignore-not-found").length() > 0) {
+			final String crds = adminBinary.execute("get", "crd", customResource, "--ignore-not-found");
+			if (crds != null && (crds.length() > 0)) {
```

Comment:
`!crds.isEmpty()` instead of length check?

Reply:
+1, thanks.